cinema_service/urls.py

```diff
@@ -5,4 +5,5 @@
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("__debug__/", include("debug_toolbar.urls")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
Ensure that the `cinema.urls` and `user.urls` modules define the `app_name` variable to match the namespaces used here. This is necessary for the namespaces to function correctly.